org.catacombing.kumo.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "org.catacombing.kumo",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm20"
+    ],
+    "command": "kumo",
+    "finish-args": [
+        "--socket=pulseaudio",
+        "--socket=wayland",
+        "--socket=pcsc",
+        "--share=network",
+        "--device=dri"
+    ],
+    "build-options": {
+        "prepend-ld-library-path": "/usr/lib/sdk/llvm20/lib",
+        "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm20/bin"
+    },
+    "modules": [{
+        "name": "unifdef",
+        "no-autogen": true,
+        "make-install-args": ["prefix=${FLATPAK_DEST}"],
+        "sources": [{
+            "type": "archive",
+            "url": "https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz",
+            "sha256": "fba564a24db7b97ebe9329713ac970627b902e5e9e8b14e19e024eb6e278d10b"
+        }],
+        "cleanup": ["*"]
+    }, {
+        "name": "woff2",
+        "buildsystem": "cmake-ninja",
+        "config-opts": [
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+            "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG"
+        ],
+        "sources": [{
+            "type": "archive",
+            "url": "https://github.com/google/woff2/archive/v1.0.2.tar.gz",
+            "sha256": "add272bb09e6384a4833ffca4896350fdb16e0ca22df68c0384773c67a175594"
+        }]
+    }, {
+        "name": "libwpe",
+        "buildsystem": "cmake-ninja",
+        "config-opts": [
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+            "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG"
+        ],
+        "sources": [{
+            "type": "archive",
+            "url": "https://wpewebkit.org/releases/libwpe-1.16.2.tar.xz",
+            "sha256": "960bdd11c3f2cf5bd91569603ed6d2aa42fd4000ed7cac930a804eac367888d7"
+        }]
+    }, {
+        "name": "wpewebkit",
+        "buildsystem": "cmake-ninja",
+        "builddir": true,
+        "config-opts": [
+            "-DPORT=WPE",
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCMAKE_C_FLAGS_RELEASE=-DNDEBUG",
+            "-DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG",
+            "-DUSE_SKIA_OPENTYPE_SVG=OFF",
+            "-DUSE_SYSPROF_CAPTURE=OFF",
+            "-DUSE_LIBBACKTRACE=OFF",
+            "-DENABLE_WPE_PLATFORM=ON",
+            "-DENABLE_BUBBLEWRAP_SANDBOX=OFF",
+            "-DENABLE_WPE_PLATFORM_DRM=OFF",
+            "-DENABLE_SPEECH_SYNTHESIS=OFF",
+            "-DENABLE_DOCUMENTATION=OFF",
+            "-DENABLE_MINIBROWSER=OFF"
+        ],
+        "build-options": {
+            "arch": {
+                "aarch64": {
+                    "cflags": "-g1",
+                    "cxxflags": "-g1"
+                }
+            }
+        },
```

Comment:
what are these for?

Reply:
This produces minimal debug info, it's copied from the official wpewebkit flatpak that they ship on their own and was previously present in the archlinux pkgbuild before switching to clang.

It's probably fine to remove it if desired, but chances are this will make building this app lighter with no major downsides for most users. We're talking about ~4GB of debug data here according to Archlinux maintainers.

Reply:
For reference: https://github.com/Igalia/wpe-flatpak/blob/master/sdk/org.wpe.Sdk.json#L218-L229

This is probably not mandatory, but I don't think it will hurt.

Reply:
the default flags are different. you shouldn't need these. We also generate debug symbols.

Reply:
default is -g which probably ends up generating gigantic debuginfo.

Reply:
> default is -g which probably ends up generating gigantic debuginfo.

I believe this would be the result, yes. I can obviously remove this, but as the developer I don't really gain much from the additional symbols so it just seems wasteful and can actually run into build failures on some systems due to the amount of symbols generated.

Reply:
why is this on aarch64 only though?

Reply:
That's a good point. Arch had it on x86 obviously, I'm not sure about flathub's builders but my aarch64 machine also struggles a bit more than my x86 one. So it's just for the platform that is most likely to fail due to symbol sizes. Since this was copied from the official wpe webkit repo, I'd assume their runners were struggling too.

I just added it for all platforms. I personally don't need it on either and helping the x86 runners a bit probably doesn't hurt.